hpi-schul-cloud/nuxt-client
# BC-820 - Design improvements school admin page

# Short Description

QA note:
Change tested on env:
https://brb-bc-820-design-improvement-admin-page.cd.dbildungscloud.dev/
https://nbc-bc-820-design-improvement-admin-page.cd.dbildungscloud.dev/
https://default-bc-820-design-improvement-admin-page.cd.dbildungscloud.dev/
No issues found. All accepted criteria from link Figma above done.
See Jira ticket: https://ticketsystem.dbildungscloud.de/browse/BC-820





## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-820


## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
